corehq/apps/commtrack/models.py

```diff
@@ -410,13 +410,14 @@ def get_display_config(cls):
 
 class ActiveManager(models.Manager):
     """
-    Filter any object that is associated to an archived product.
+    Filter any object that is associated to an archived product, or deleted location.
     """
 
     def get_queryset(self):
         return super(ActiveManager, self).get_query_set() \
             .exclude(sql_product__is_archived=True) \
-            .exclude(sql_location__is_archived=True)
+            .exclude(sql_location__is_archived=True) \
+            .exclude(sql_location=None)
```

Comment:
Oh, I just saw your comment below - looks like we're already on the same page here, feel free to disregard. :smile: